### PR TITLE
ppx: textarea tag should be allowed to have no child 

### PR DESCRIPTION
**Pull request by ghuysmans**
The solution relies on previous work for empty `<script>` tags.

### CODE REVIEW

File: syntax/element_content.ml [hunk 1]
```diff
@@ -228,6 +228,8 @@ let script ~lang ~loc ~name children =
     [Nolabel, child]
   | _ -> Common.error loc "%s can have at most one child" name
 
+let textarea = script
```
Comment (Drup) [inline syntax/element_content.ml:231]: I would prefer you pulled the function as a `at_most_one_children` function, and alias that both time. :)

File: syntax/element_content.ml [hunk 2]
```diff
@@ -228,6 +228,8 @@ let script ~lang ~loc ~name children =
...
+
```
Comment (Drup) [inline syntax/element_content.ml:232]: A link to the spec in comment would be nice too.

**Review by Drup — APPROVED** [review]
Hi, thanks for the contribution. It looks good, I'll let you solve the small comments and merge! :)

File: syntax/element_content.ml (hunk 1 above)
Comment (ghuysmans) [reply syntax/element_content.ml]: Oh, right, that's way clearer.

File: syntax/element_content.ml (hunk 2 above)
Comment (ghuysmans) [reply syntax/element_content.ml:235]: Done; too bad I can't link the _Content model_ part.

**Comment by Drup** [comment]
Oh, didn't realized you had merge rights. Thanks ! :)

**Comment by ghuysmans** [comment]
Thank you for your review :)

**Pull request merged**